src/data.rs

```diff
@@ -220,11 +220,12 @@ pub struct PlatformMemory {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Clone)]
 pub struct PlatformMemory {
```

Comment:
This doesn't cover all 24 items in [`vm_statistics64`](https://docs.rs/libc/latest/x86_64-apple-darwin/libc/struct.vm_statistics64.html). Does it need to?

Reply:
I don't think so? That struct seems to include a lot of… well, statistics — like faults/pageins/pageouts whatever that are not related to *current usage*.

Reply:
Ok. I've added most of the types that look like they are not statistics-oriented.